src/jvm/main/org/jetbrains/kotlinx/lincheck/TraceDebugger.kt

```diff
@@ -10,4 +10,81 @@
 
 package org.jetbrains.kotlinx.lincheck
 
+import org.jetbrains.kotlinx.lincheck.execution.ExecutionResult
+import org.jetbrains.kotlinx.lincheck.execution.ExecutionScenario
+import org.jetbrains.kotlinx.lincheck.execution.threadsResults
+import org.jetbrains.kotlinx.lincheck.strategy.managed.forClasses
+import org.jetbrains.kotlinx.lincheck.strategy.managed.modelchecking.ModelCheckingOptions
+import org.jetbrains.kotlinx.lincheck.transformation.LincheckJavaAgent.ensureClassHierarchyIsTransformed
+import org.jetbrains.kotlinx.lincheck.verifier.Verifier
+import java.lang.reflect.Method
+import java.lang.reflect.Modifier
+import kotlin.reflect.jvm.kotlinFunction
+
 val isInTraceDebuggerMode = System.getProperty("lincheck.traceDebuggerMode", "false").toBoolean()
+val classUnderTraceDebugging: String = System.getProperty("traceDebugger.className", "")
```

Comment:
Please move these properties to `TraceDebuggerAgent.kt` -- they are part of the agent configuration.